inc/saipoe.h

```diff
@@ -0,0 +1,524 @@
+/**
+ * Copyright 2024 Marvell.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *    not use this file except in compliance with the License. You may obtain
+ *    a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#if !defined (__SAIPOE_H_)
+#define __SAIPOE_H_
+
+#include <saitypes.h>
+
+/** poe device power limit mode
+ *  port limit - max power limit per port is configured by the user 
+ *  class limit - max power is set automatically according the class of the connected device
+ */
+typedef enum _sai_poe_device_type_limit_mode_t
+{
+    SAI_POE_DEVICE_LIMIT_MODE_TYPE_PORT_LIMIT = 0,
```

Comment:
SAI\_POE\_DEVICE\_TYPE\_LIMIT\_MODE\_PORT\_LIMIT instead of SAI\_POE\_DEVICE\_LIMIT\_MODE\_TYPE\_PORT\_LIMIT

Reply:
Done, thanks.